<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Identification</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image-icon.qrc">
    <normaloff>:/image/userpassword_deusuari_787.png</normaloff>:/image/userpassword_deusuari_787.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="topMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">image: url(:/image/index.jpeg);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="verticalWidget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>40</number>
              </property>
              <property name="bottomMargin">
               <number>40</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(156, 84, 84);
font-size:26px;
font-weight:bold;</string>
                </property>
                <property name="text">
                 <string>          Gestion de stock </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(156, 84, 84);
font-size:24px;
font-weight:bold;</string>
                </property>
                <property name="text">
                 <string>           de pi√®ce de rechange</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(104, 10, 10);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
font-size:20px;
font-weight:bold;</string>
             </property>
             <property name="text">
              <string>                       IDENTIFICATION</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(:/image/people_114360.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="formWidget" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="verticalSpacing">
               <number>15</number>
              </property>
              <property name="rightMargin">
               <number>40</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditNomUtilisateur">
                <property name="styleSheet">
                 <string notr="true">QLineEdit{
	
	background-color: rgb(104, 10, 10);
	color: white;
	border:none;
	font-size:18px;
	border-bottom:2px solid  white;
	padding:2px 2px 5px 12px;
}
QLineEdit:focus{
border-bottom:3px solid  white;
	
}
</string>
                </property>
                <property name="placeholderText">
                 <string>Nom d'utilisateur</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditPasswdUtilisateur">
                <property name="styleSheet">
                 <string notr="true">QLineEdit{
	background-color: rgb(104, 10, 10);
	color: white;
	border:none;
	font-size:18px;
	border-bottom:2px solid  white;
	padding:2px 2px 5px 12px;
}
QLineEdit:focus{
border-bottom:3px solid  white;
	
}
</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Mot de passe</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="comboBox">
                <property name="styleSheet">
                 <string notr="true">#comboBox {
	color: rgb(255, 255, 255);
background-color: rgb(94, 12, 12);
}
#comboBox:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(156, 84, 84);

}</string>
                </property>
                <item>
                 <property name="text">
                  <string>Type </string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Administrateur</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Utilisateur</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="leftMargin">
               <number>40</number>
              </property>
              <property name="rightMargin">
               <number>40</number>
              </property>
              <property name="bottomMargin">
               <number>50</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QPushButton" name="btnIdentifier">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#btnIdentifier{
	background-color: rgb(104, 10, 10);
	font: 700 12pt &quot;Serif&quot;;
	color: white;
	border-radius:10px;
	border: 2px solid white;
}
#btnIdentifier:hover{
	background-color: white;
	font: 700 12pt &quot;Serif&quot;;
	color: rgb(104, 10, 10);
	border-radius:10px;

}
</string>
                  </property>
                  <property name="text">
                   <string>IDENTIFIER</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image-icon.qrc"/>
 </resources>
 <connections/>
</ui>
